src/components/TransactionDetails.tsx

```diff
@@ -86,9 +84,24 @@ export default function TransactionDetails({
           {isLoading ? (
             <SkeletonLoader rows={7} screen={SkeletonLoaderScreen.Tx} />
           ) : (
-            transactions.map((item) => (
-              <TransactionRow key={item.hash} rawData={item} />
-            ))
+            transactions.map((item) => {
+              const transformedData = transformTransactionData(item);
+              return (
+                <TransactionRow
+                  key={item.hash}
+                  data={{
+                    transactionType: transformedData.transactionType,
+                    from: transformedData.from,
+                    to: transformedData.to,
+                    hash: transformedData.hash,
+                    amount: transformedData.amount,
+                    symbol: transformedData.symbol,
+                    timeInSec: transformedData.timeInSec,
+                    status: transformedData.status,
+                  }}
+                />
+              );
+            })
           )}
```

Comment:
```suggestion
            transactions.map((item) => (
                <TransactionRow
                  key={item.hash}
                  data={transformTransactionData(item)}
                />
              )
            )
          )}
```

Reply:
type is not the same as what `transformTransactionData` returns 😅